docs/CONFIGURATION.md

```diff
@@ -186,6 +186,7 @@ driven operation.
 
 Parameters that can be used with any input plugin:
 
+- **alias**: Alias that will get logged along with the plugin name. Allows to differentiate between multiple definitions of a plugin.
```

Comment:
Split this definition from the rest of the PR, this can go into 1.12.1 and the rest into 1.13.

The documentation should describe `alias` as a way to name the instance of the plugin, you can mention logging and internal metrics as examples of where this may appear, but don't describe it as just a logging option.

Also line wrap at 78-80 chars.

Reply:
Why would we document it before it's supported though? Seems like a package deal.